### PR TITLE
Fixed issue #17195: When attaching PDF to email invitation, full path  is kept as value of a hidden input

### PR DESCRIPTION
**Pull request by gabrieljenik**
Changing from full path to relative path on that input

### CODE REVIEW

**Comment by Shnoulle** [comment]
I'm not usre i undersand the point here ?

My opinion : we currenl-ty save total path in survey : /var/www/upload/survey/1234/files/myfile.pdf

Then : 
1. If we move LSS + ressources on another server : file is lost
2. If we copy survey to 12345 : it take old file (fro 1234 survey), but when delete 1234 : file was deleted.

My opinion : we must save and use only `myfile.pdf` or `subdir/myfile.pdf`

**Comment by Shnoulle** [comment]
ok, same for theme editor then :)

File: application/controllers/admin/emailtemplates.php [hunk 1]
```diff
@@ -63,10 +63,19 @@ function index($iSurveyId)
...
             $sEscapeMode = 'unescaped';
         }
+        $uploadDir = realpath(Yii::app()->getConfig('uploaddir'));
         foreach ($grplangs as $key => $grouplang) {
             $aData['bplangs'][$key] = $grouplang;
             $aData['attrib'][$key] = SurveyLanguageSetting::model()->find('surveyls_survey_id = :ssid AND surveyls_language = :ls', array(':ssid' => $iSurveyId, ':ls' => $grouplang));
-            $aData['attrib'][$key]['attachments'] = unserialize($aData['attrib'][$key]['attachments']);
+            $attachments = unserialize($aData['attrib'][$key]['attachments']);
+            foreach ($attachments as &$template) {
+                foreach ($template as &$attachment) {
```
Comment (c-schmitz) [inline application/controllers/admin/emailtemplates.php:72]: PHP Error [500]

Invalid argument supplied for foreach() (C:\xampp\htdocs\limesurvey400\application\controllers\admin\emailtemplates.php:71)

#0 C:\xampp\htdocs\limesurvey400\framework\base\CApplication.php(834): CErrorHandler->handle()
#1 C:\xampp\htdocs\limesurvey400\application\controllers\admin\emailtemplates.php(71): LSYii_Application->handleError()
#2 unknown(0): emailtemplates->index()
#3 C:\xampp\htdocs\limesurvey400\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#4 C:\xampp\htdocs\limesurvey400\application\core\Survey_Common_Action.php(86): emailtemplates->runWithParamsInternal()
#5 C:\xampp\htdocs\limesurvey400\framework\web\CController.php(308): emailtemplates->runWithParams()
#6 C:\xampp\htdocs\limesurvey400\framework\web\CController.php(286): AdminController->runAction()
#7 C:\xampp\htdocs\limesurvey400\framework\web\CController.php(265): AdminController->runActionWithFilters()
#8 C:\xampp\htdocs\limesurvey400\application\controllers\AdminController.php(202): AdminController->run()
#9 C:\xampp\htdocs\limesurvey400\framework\web\CWebApplication.php(282): AdminController->run()
#10 C:\xampp\htdocs\limesurvey400\framework\web\CWebApplication.php(141): LSYii_Application->runController()
#11 C:\xampp\htdocs\limesurvey400\framework\base\CApplication.php(185): LSYii_Application->processRequest()
#12 C:\xampp\htdocs\limesurvey400\index.php(182): LSYii_Application->run()

**Review by c-schmitz — CHANGES_REQUESTED** [review]

File: application/controllers/admin/emailtemplates.php (hunk 1 above)
Comment (gabrieljenik) [reply application/controllers/admin/emailtemplates.php]: Should be fixed now

**Pull request merged**